examples/basic/main.tf

```diff
@@ -0,0 +1,12 @@
+provider "digitalocean" {
+  token = var.do_token
+}
+# Create a new Web Droplet in the nyc1 region
+
+module "digitalocean_droplet" {
+  source = "github.com/opsd-io/terraform-module-digitalocean-droplet?ref=MB_droplet_creation"
```

Comment:
Set ref to `main`.
For local development use solution like this: https://github.com/opsd-io/terraform-module-digitalocean-vpc/blob/main/examples/create_vpc/override.tf

Reply:
changed to main as this will be correct after merge . thanks